source/administration/announcement-banner.rst

```diff
@@ -0,0 +1,20 @@
+Announcement Banner
+===================
+
+In Mattermost Enterprise Edition, an announcement banner is available that allows system administrators to display a notice that's visible to all users on the system.
+
+.. image:: ../images/announcement-banner-1106x272.png
+  :width: 1106
+  :height: 272
+  :alt: Shows the announcement banner at the top of the user's screen.
+
+By default, users can dismiss the banner until you either change the text of the banner or until you re-activate the banner. A setting in the System Console allows you to prevent users from dismissing the banner. You can also control the text color and the background color.
```

Comment:
"until you either change the text of the banner or until you re-activate the banner" > We should just change this to state that the admin can chose to make the banner permanent (can't be dismissed), if set in the console.